modules/artifact/src/main/java/org/mule/runtime/module/artifact/classloader/CompositeClassLoader.java

```diff
@@ -87,12 +96,13 @@ else if (lookupStrategy == PARENT_FIRST)
             }
             catch (ClassNotFoundException e)
             {
-                result = doLoadClass(name);
+                exceptions.add(e);
```

Comment:
Not sure that this info will be helpful. Each composed classloader will be following the path up to the container classloader. So, I guess the list of exceptions will contain multiple occurrences of a classnotfound on the container classloader